### ISSUE TITLE
Justice Screen Filters Stop Working - DFHack 50.11-r4

### ISSUE THREAD

**Issue by Dragoon209** [DFHack/dfhack#4069]
I have had the filters break several times during a game session, but I haven't found the cause yet.  The manifestation is that changing the filter (Citizens, Risky visitors, etc)  no longer filters the list.  The entire unaltered list is presented.

Additionally, I have had a few instances where after activating search, keypresses are still passed to the game as well as the search bar.

Thanks!

**Comment by myk002** [comment]
I'll take a look. Thanks!

For the second issue (which may have been better as a separate GitHub issue) where keypresses are still passed to the game as well as the search bar, which screen do you have open for the search?

**Comment by myk002** [comment]
Could you possibly try the `testing` branch on Steam and see if this is still reproducible for you? I can't seem to get either of the symptoms to trigger with current code.

**Comment by myk002** [comment]
I've tried some more and have been unable to reproduce the problem. If you do run into this again, could you reopen this issue?

**Comment by Dragoon209** [comment]
I played around with this some more, and it looks like it was another
plugin interfering. MRAU was enabled in overlays, but not engaged. I'm not
sure yet how to manually trigger it, but eventually it causes the symptoms
I've described.

After disabling it in overlays, I have not gotten the problem to reoccur.
Thanks for looking in to this, and sorry for the false positive!  Should
plugins be able to interact with each other in this fashion?
